### PR TITLE
[Fix] Fix optional issues in docstring

### PR DESCRIPTION
**Pull request by LXXXXR**
Hi,
It has come to my knowledge that 'optional' should only be used when indicating None is an acceptable type. So this pull request fixes the optional issue in docstring. 
Pls refer to the following link on the issue.
https://stackoverflow.com/questions/34846731/where-can-i-find-more-information-about-new-syntax-supported-in-google-style-doc

Thx.

### CODE REVIEW

File: mmcls/models/losses/focal_loss.py [hunk 1]
```diff
@@ -93,8 +92,8 @@ def forward(self,
             (N, *). Dafaults to None.
             avg_factor (int, optional): Average factor that is used to average
             the loss. Defaults to None.
-            reduction_override (str, optional): The method used to reduce the
-                loss into a scalar. Options are "none", "mean" and "sum".
+            reduction_override (str): The method used to reduce the loss into
+                a scalar. Options are "none", "mean" and "sum".
```
Comment (MeowZheng) [inline mmcls/models/losses/focal_loss.py:96]: reduction_overr type can be optional

Comment (LXXXXR) [reply mmcls/models/losses/focal_loss.py]: My bad

File: mmcls/models/backbones/shufflenet_v1.py [hunk 2]
```diff
@@ -22,20 +22,20 @@ class ShuffleUnit(nn.Module):
...
             convolutions in each ShuffleUnit. Default: 3
-        first_block (bool, optional): Whether it is the first ShuffleUnit of a
+        first_block (bool): Whether it is the first ShuffleUnit of a
             sequential ShuffleUnits. Default: False, which means not using the
             grouped 1x1 convolution.
-        combine (str, optional): The ways to combine the input and output
+        combine (str): The ways to combine the input and output
             branches. Default: 'add'.
         conv_cfg (dict): Config dict for convolution layer. Default: None,
             which means using conv2d.
```
Comment (MeowZheng) [inline mmcls/models/backbones/shufflenet_v1.py:33]: add optional

File: mmcls/models/backbones/shufflenet_v1.py [hunk 3]
```diff
@@ -154,9 +154,9 @@ class ShuffleNetV1(BaseBackbone):
     """ShuffleNetV1 backbone.
 
     Args:
```
Comment (MeowZheng) [inline mmcls/models/backbones/shufflenet_v1.py:156]: conv_cfg add optional

**Review by MeowZheng — APPROVED** [review]

**Review by ycxioooong — APPROVED** [review]
LGTM
